modules/sc-mesh-secure-deployment/src/nats/initd/S90mdm_agent

```diff
@@ -18,9 +18,9 @@ BC_CERT_FILE=$(find /opt/crypto/rsa/birth/filebased/DNS/ -type f -name '*.local.
 BC_CA_FILE="/opt/mspki/rsa/certificate_chain.crt"
 
 if [ -e "$BC_CA_FILE" ] && [ -e "$BC_KEY_FILE" ] && [ -e "$BC_CERT_FILE" ]; then
-  command_args="-k $BC_KEY_FILE -c $BC_CERT_FILE -r $BC_CA_FILE"
+  command_args="-k $BC_KEY_FILE -c $BC_CERT_FILE -r $BC_CA_FILE -i bat0"
```

Comment:
This will indeed cause that MDM agent tries to monitor MDM server via bat0 interface. However, there are other issues that effectively prevents it from functioning.  MDM agent starts sevice monitoring for the wanted interface when interface status is UP. This doesn't work currently with Batman interfaces as they seem to remain in UNKNOWN state. Yet another issue is that MDM agent tries to stop monitoring only when interface state becomes DOWN and that doesn't work with for example LAN1 when one detaches cable. I am already working on those other issues.